.github/scripts/tryrebase.py

```diff
@@ -27,6 +27,28 @@ def parse_args() -> Any:
     return parser.parse_args()
 
 
+def post_already_uptodate(
+    pr: GitHubPR, repo: GitRepo, onto_branch: str, dry_run: bool
+) -> None:
+    msg = f"Tried to rebase and push PR #{pr.pr_num}, but it was already up to date."
+    def_branch = pr.default_branch()
+    def_branch_fcn = f"refs/remotes/{repo.remote}/{def_branch}"
+    if onto_branch != def_branch_fcn and repo.rev_parse(
+        def_branch_fcn
+    ) != repo.rev_parse(onto_branch):
+        def_branch_url = f"https://github.com/{pr.org}/{pr.project}/tree/{def_branch}"
+        msg += f" Try rebasing against [{def_branch}]({def_branch_url}) by issuing:"
+        msg += f"\n`@pytorchbot rebase -b {def_branch}`"
```

Comment:
I'm curious, what's the point of this message? Is it guessing that the user accidentally requested a rebase against the non-default branch?